### ISSUE TITLE
Support multi-cultural emoji display formats

### ISSUE THREAD

**Issue by n8fr8** [lolibrarian/NYPL-Emoji-Bot#8]
Message from a friend, who is not blonde... let's make this happen!

Technically, you just need to look for the "combining characters" Apple tags on the end to indicate the skin tone of the emoji. All the technical detail is here: http://www.unicode.org/reports/tr51/tr51-2.html#Diversity

"So I just tried a flamenco dancer. I picked one out of the many (recently introduced variations) to account for my skin tone and dark hair. I also figured it might generate more accurate images given the roots of flamenco.

When @NYPLenojibot responded, it returned a lovely photograph. Not, admittedly of an authentic flamenco setting, but of famous American danseuse Pauline Koner (born in New York to Russian immigrants) “...wearing a flamenco dress performing a Spanish dance. She appears to be outdoors.”

So why does this matter? I was sort of surprised that the response contained not just the image, but the emoji I originally tweeted (presumably as a tag): except the one it returned was fair skinned and blonde. 

I try not to turn tech issues (or indeed any issues) into racial bias issues unless they’re really glaring, but it struck me that this was insensitive. If your bot is an _emoji_ bot, surely it can cater for the range of variations that have been introduced to address race, skin tone, etc.? Or in other words, All Of The Emojis."

**Comment by mgiraldo** [comment]
i'd rephrase this to: “have the bot return the same emoji in the tweet as it was submitted”

we need to see if there is a NodeJS module that allows for detection of skin variation in the emoji but considering [this article](https://medium.com/@mroth/how-i-kept-building-emojitracker-c31378810136#.cp56pe30z) it seems unlikely for now

**Comment by lolibrarian** [comment]
Hi @n8fr8, thanks for opening this issue! Needs more than 140 characters to cover. 😄 

Emoji encoding, as @mgiraldo notes above, is complicated. All images are internally filed under the yellow base character: up til now, we simply replied to any tweet with the character it was filed under. 

We are about to push a fix that allows for the bot to reply with the skin tone of the request tweet.

Thanks again for flagging this! Have fun with the bot 🚀 

 —Lauren
